docs/common/migrations.md

```diff
@@ -29,9 +29,11 @@ These SQL statements are run by the `Database.Schema.migrate()` method. Migratio
 
 ## Verifying Migrations
 
-You can also place a `.db` file in the `src/main/sqldelight` folder of the same `<version number>.db` format. If there is a `.db` file present, a new `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task, meaning your migrations will be verified against that `.db` file. It confirms that the migrations yield a database with the latest schema.
+A `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task. For any `.db` file named `<version number>.db` in your SqlDelight source set (e.g. `src/main/sqldelight`) it will apply all migrations starting from `<version number>.sqm`. and confirms that the migrations yield a database with the latest schema.
 
-To generate a `.db` file from your latest schema, run the `generateSqlDelightSchema` task, which is available once you specify a `schemaOutputDirectory`, as described in the [gradle.md](gradle.md). You should probably do this before you create your first migration.
+To generate a `.db` file from your latest schema, run the `generate<database name>Schema` task, which is available once you specify a `schemaOutputDirectory`, as described in the [gradle.md](gradle.md). You should probably do this before you create your first migration.
```

Comment:
There is no `generateSqlDelightSchema` registered anymore (not sure if that is a bug or not).

On Android the task is `generate<variant><database name>Schema` but not sure if that should be called out (or what is generated for other targets).